hawkbit-repository/hawkbit-repository-jpa/src/test/java/org/eclipse/hawkbit/repository/jpa/ControllerManagementTest.java

```diff
@@ -678,6 +680,67 @@ private void updateAttributeAndVerify(final String controllerId) {
                 .isEqualTo(testData);
     }
 
+    @Test
```

Comment:
keep one test method but move different test cases (for different update modes) into separate methods which are annotated with Step annotation